packages/flutter/test/material/card_test.dart

```diff
@@ -188,4 +188,38 @@ void main() {
     }));
     expect(tester.widget<Material>(find.byType(Material)).clipBehavior, Clip.antiAliasWithSaveLayer);
   });
+
+  testWidgets('Card shadowColor', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const Card(),
+    );
+
+    expect(_getCard(tester).shadowColor, null);
+    expect(_getCardMaterial(tester).shadowColor, const Color(0xFF000000));
+
+    await tester.pumpWidget(
+      const Card(
+        shadowColor: Colors.red,
+      ),
+    );
+
+    expect(_getCardMaterial(tester).shadowColor, _getCard(tester).shadowColor);
+    expect(_getCardMaterial(tester).shadowColor, Colors.red);
+  });
+}
+
+
+Material _getCardMaterial(WidgetTester tester) {
+  return tester.widget<Material>(
+    find.descendant(
+      of: find.byType(Card),
+      matching: find.byType(Material),
+    ),
+  );
+}
+
+Card _getCard(WidgetTester tester) {
```

Comment:
I think by convention Flutter usually puts helper functions like these at the top of the test file, like just inside of `main`.  Also, there shouldn't be any double line breaks like lines 210-211.

Reply:
Sure I'll fix that up.

Reply:
Awesome, thanks for the quick response.  I want to get someone else to look at this that knows more about some upcoming theming changes.  It will probably be slow because of the holidays, but I'll be sure to follow up.